docs/assets/scss/_syntax.scss

```diff
@@ -1,63 +1,63 @@
-.hll { background-color: #ffc; }
+.hll { background-color: #ffffcc }
 .c { color: #999; }
-.err { color: #a00; background-color: #faa; }
-.k { color: #069; }
-.o { color: #555; }
+.err { color: #AA0000; background-color: #FFAAAA }
+.k { color: #006699; }
+.o { color: #555555 }
 .cm { color: #999; }
-.cp { color: #099; }
+.cp { color: #009999 }
 .c1 { color: #999; }
 .cs { color: #999; }
-.gd { background-color: #fcc; border: 1px solid #c00; }
-.ge { font-style: italic; }
-.gr { color: #f00; }
-.gh { color: #030; }
-.gi { background-color: #cfc; border: 1px solid #0c0; }
-.go { color: #aaa; }
-.gp { color: #009; }
-.gu { color: #030; }
-.gt { color: #9c6; }
-.kc { color: #069; }
-.kd { color: #069; }
-.kn { color: #069; }
-.kp { color: #069; }
-.kr { color: #069; }
-.kt { color: #078; }
-.m { color: #f60; }
-.s { color: #d44950; }
-.na { color: #4f9fcf; }
-.nb { color: #366; }
-.nc { color: #0a8; }
-.no { color: #360; }
-.nd { color: #99f; }
-.ni { color: #999; }
-.ne { color: #c00; }
-.nf { color: #c0f; }
-.nl { color: #99f; }
-.nn { color: #0cf; }
+.gd { background-color: #FFCCCC; border: 1px solid #CC0000 }
+.ge { font-style: italic }
+.gr { color: #FF0000 }
+.gh { color: #003300; }
+.gi { background-color: #CCFFCC; border: 1px solid #00CC00 }
+.go { color: #AAAAAA }
+.gp { color: #000099; }
+.gu { color: #003300; }
+.gt { color: #99CC66 }
+.kc { color: #006699; }
+.kd { color: #006699; }
+.kn { color: #006699; }
+.kp { color: #006699 }
+.kr { color: #006699; }
+.kt { color: #007788; }
+.m { color: #FF6600 }
+.s { color: #d44950 }
+.na { color: #4f9fcf }
+.nb { color: #336666 }
+.nc { color: #00AA88; }
+.no { color: #336600 }
+.nd { color: #9999FF }
+.ni { color: #999999; }
+.ne { color: #CC0000; }
+.nf { color: #CC00FF }
+.nl { color: #9999FF }
+.nn { color: #00CCFF; }
 .nt { color: #2f6f9f; }
-.nv { color: #033; }
-.ow { color: #000; }
-.w { color: #bbb; }
-.mf { color: #f60; }
-.mh { color: #f60; }
-.mi { color: #f60; }
-.mo { color: #f60; }
-.sb { color: #c30; }
-.sc { color: #c30; }
-.sd { font-style: italic; color: #c30; }
-.s2 { color: #c30; }
-.se { color: #c30; }
-.sh { color: #c30; }
-.si { color: #a00; }
-.sx { color: #c30; }
-.sr { color: #3aa; }
-.s1 { color: #c30; }
-.ss { color: #fc3; }
-.bp { color: #366; }
-.vc { color: #033; }
-.vg { color: #033; }
-.vi { color: #033; }
-.il { color: #f60; }
+.nv { color: #003333 }
+.ow { color: #000000; }
+.w { color: #bbbbbb }
+.mf { color: #FF6600 }
+.mh { color: #FF6600 }
+.mi { color: #FF6600 }
+.mo { color: #FF6600 }
+.sb { color: #CC3300 }
+.sc { color: #CC3300 }
+.sd { color: #CC3300; font-style: italic }
+.s2 { color: #CC3300 }
+.se { color: #CC3300; }
+.sh { color: #CC3300 }
+.si { color: #AA0000 }
+.sx { color: #CC3300 }
+.sr { color: #33AAAA }
+.s1 { color: #CC3300 }
+.ss { color: #FFCC33 }
+.bp { color: #336666 }
+.vc { color: #003333 }
+.vg { color: #003333 }
```

Comment:
Color `#003333` should be written as `#033`<br>Declaration should be terminated by a semicolon